4q.txt

```diff
@@ -0,0 +1,26 @@
+import csv
+#CSV FILE
+data = [
+    ["sid", "sname", "salary"],
+    [1, "savan", 3800],
+    [2, "om", 4000],  
+    [3, "sahu", 4500],
+    [4, "hiren", 4100],
+    [5, "sai", 4200]   
+]
+#OPEN A CSV FILE
+with open('salary.csv', 'w', newline='') as f:

```

Comment:
newline=' '..why we have to write it?

Reply:
empty list are autometcally insted if dont use "new line".it will create problem for if condition.